dataform/examples/dataform_udf_unit_test/dataform.json

```diff
@@ -0,0 +1,6 @@
+{
+  "warehouse": "bigquery",
+  "defaultSchema": "fn",
+  "defaultDatabase": "bqutil",
```

Comment:
Should these values be templated or placeholders instead of pointing to bqutil?

Reply:
These must stay as-is since the example executes the unit tests for the bqutil UDFs